coriolis/utils.py

```diff
@@ -274,12 +274,15 @@ def exec_ssh_cmd(ssh, cmd, environment=None, get_pty=False):
             "Command \"%s\" failed with exit code: %s\n"
             "stdout: %s\nstd_err: %s" %
             (cmd, exit_code, std_out, std_err))
+    if get_pty:
```

Comment:
The python ```splitlines()``` function does a good job in identifying the newline termination properly:

```python
>>> data = "this is \r\ntest \n with carriage return"
>>> print(data)
this is 
test 
 with carriage return
>>> print(data.splitlines())
['this is ', 'test ', ' with carriage return']
```

Do we need to replace ```\r\n``` with ```\n``` ?